soda_machine.rb

```diff
@@ -7,12 +7,17 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
```

Comment:
Nothing wrong here, just pointing out that because of the attr_reader you could drop the @ here (and future uses of @sodas)